Userland/Libraries/LibC/stdlib.cpp

```diff
@@ -671,6 +672,23 @@ void srand(unsigned seed)
     s_next_rand = seed;
 }
 
+// https://pubs.opengroup.org/onlinepubs/9699919799/functions/drand48.html
+double drand48()
+{
+    static const uint64_t a = 0x5DEECE66DULL;
+    static const uint64_t c = 0xBULL;
+    static const uint64_t m = 1ULL << 48;
+
+    s_next_rand48 = (a * s_next_rand48 + c) & (m - 1);
+    return (double)s_next_rand48 / m;
+}
+
+// https://pubs.opengroup.org/onlinepubs/9699919799/functions/srand48.html
+void srand48(long seed)
+{
+    s_next_rand48 = (seed << 16) | 0x330E;
```

Comment:
It probably doesn't matter but you're doing more than what the standard is asking. You're supposed to set the 32 high-order bits (so from 16 to 48) but you also set things above the 48 first bits